components/list/list-item.js

```diff
@@ -0,0 +1,183 @@
+				margin-right: 0rem;
+			}
+			::slotted([slot="illustration-outer"]) {
+				margin-bottom: 18px;
+				margin-top: 18px;
+			}
+			::slotted([slot="actions"]) {
+				align-self: flex-start;
+				display: flex;
+				flex-grow: 0;
+			}
+			.d2l-list-item-main {
+				width: 100%;
+			}
+		`;
+		const mainContent = css`
+			::slotted(.d2l-list-item-text) {
+				margin: 0;
+				max-height: 2.4rem;
+				overflow: hidden;
+			}
+			::slotted(.d2l-list-item-text-secondary-responsive),
+			::slotted(.d2l-list-item-text-secondary) {
+				margin: 0;
+				margin-top: 0.3rem;
+				overflow: hidden;
+			}
+			::slotted(.d2l-list-item-text-secondary-responsive) {
+				display: none;
+			}
+		`;
+		const divider = css`
+			.d2l-list-item-divider-top,
+			.d2l-list-item-divider-bottom {
+				border: 0;
+				border-top: 1px solid var(--d2l-color-mica);
+				margin: 0;
+				position: absolute;
+				width: 100%;
+			}
+			.d2l-list-item-divider-top {
+				display: var(--d2l-list-item-divider-top, none);
+				top: -1px;
+			}
+			.d2l-list-item-divider-bottom {
+				bottom: -1px;
+				display: var(--d2l-list-item-divider-bottom, none);
+			}
+		`;
+
+		const breakPoint580 = css`
+			.d2l-list-item-flex[breakpoint="580"] ::slotted([slot|="illustration"]) {
+				margin-right: 1rem;
+				max-height: 71px;
+				max-width: 120px;
+			}
+			:host([dir="rtl"]) .d2l-list-item-flex[breakpoint="580"] ::slotted([slot|="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			.d2l-list-item-flex[breakpoint="580"] ::slotted(.d2l-list-item-text-secondary-responsive) {
+				display: block;
+			}
+		`;
+
+		const breakPoint636 = css`
+			.d2l-list-item-flex[breakpoint="636"] ::slotted([slot|="illustration"]) {
+				margin-right: 1rem;
+				max-height: 102px;
+				max-width: 180px;
+			}
+			:host([dir="rtl"]) .d2l-list-item-flex[breakpoint="636"] ::slotted([slot|="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			.d2l-list-item-flex[breakpoint="636"] ::slotted(.d2l-list-item-text-secondary-responsive) {
+				display: block;
+			}
+		`;
+
+		const breakPoint842 = css`
+			.d2l-list-item-flex[breakpoint="842"] ::slotted([slot|="illustration"]) {
+				margin-right: 1rem;
+				max-height: 120px;
+				max-width: 216px;
+			}
+			:host([dir="rtl"]) .d2l-list-item-flex[breakpoint="842"] ::slotted([slot|="illustration"]) {
+				margin-left: 1rem;
+				margin-right: 0;
+			}
+			.d2l-list-item-flex[breakpoint="842"] ::slotted(.d2l-list-item-text-secondary-responsive) {
+				display: block;
+			}
+		`;
+		return [ layout, mainContent, divider, breakPoint580, breakPoint636, breakPoint842];
+	}
+
+	constructor() {
+		super();
+		this._breakpoint = 0;
+		this._breakpointList = [842, 636, 580, 0];
+	}
+
+	render() {
+		return html`
+			<div class="d2l-list-item-flex" breakpoint="${this._breakpoint}">
+				<slot name="illustration-outer"></slot>
+				<div class="d2l-list-item-content">
+					<hr class="d2l-list-item-divider-top"></hr>
```

Comment:
Using `hr` to draw the dividers so I can position them properly in between other items. This will also allow them to work on hover. See last demo.

Reply:
`hr`s are meant for "thematic breaks" between paragraphs... like picture the three dots or lines you see in a novel when a story shifts scenes. A screenreader will actually call them out, which in this case I don't think we want.

I'd recommend just using the top/bottom border of the list items themselves -- I think you can accomplish all the variations you need:
* none: no borders anywhere
* all: top border on the first list item, bottom border on all list items
* middle/between: no border on the list, bottom border on all list items except the last one

Another advantage to this approach is 2 less DOM nodes per item. :)

Reply:
That is true and I want to try something to make the border work even though I am pretty sure I have tried it.

The main issue is adding a border makes it harder to overlay the borders. Since the top of one needs to be the same pixels as the bottom of the one above it. In my head this should be accomplished by what you are saying but I also tried not using the `<hr>` (which can be switched to divs in the worst case) and using borders and it didn't work.

Going to try it again though because this is the better way to go about it. Also, it is dragon slayer so it sort of fits.

Reply:
GOOD NEWS! I got it to work. Say good bye the `hr` and extra dom. CSS is a little more messy but not too bad.